app/utils/tracking.ts

```diff
@@ -0,0 +1,41 @@
+declare const ga: any;
+declare const fbq: any;
+declare const twttr: any;
+import { Web3Type } from "../actions/constants";
+import config from "../config";
+
+export const trackCommitEvent = (amount: string, web3Type: Web3Type): void => {
+  try {
+    ga("send", "event", "commit", web3Type.toString(), amount);
+
+    fbq("trackCustom", "GenericSignUp", {
+      value: amount,
+    });
+
+    twttr.conversion.trackPid(config.contractsDeployed.twitterTrackCommitId, {
```

Comment:
i think that all of these calls can fail if there is a adblocker blocking loading of scripts. Even though we have try catch here I would prefere to have explicit if like `if (twttr) { ...` or something along the lines.